website/content/specification/syntax/constraints.md

```diff
@@ -202,6 +202,20 @@ The `index-has-key` constraint is a type of Metaschema constraint that cross-ref
 The `@name` flag of an `<index>` constraint MUST specify the name of a previously defined `index` constraint.
 
 The `index-has-key` constraint has the same flags and assemblies as a [`index`](#index-constraints) constraint.
+## `is-unique` constraints
+
+The `is-unique` constraint is a type of Metaschema constraint that checks that a computed key, based on field and flag values, does not occur more than once.
+
+The [`id`](#id) attribute of an `is-unique` constraint specifies an identifier for the constraint.
+
+The `@target` attribute of an `<is-unique>` constraint identifies the node(s) in a document instance that are checked for uniqueness based on the computed key. The `is-unique` constraint MUST define a [`@target`](#target) with a Metapath expression.
+
+The `key-field` field of an `<is-unique>` constraint defines the attribute or element value that is the key for each entry in the index. The `<key-field>` element MUST define a [`@target`](#target) attribute with a Metapath expression evaluated relative to [the evaluation focus](#constraint-processing) of each entry of the matches of the constraint's `@target`.
+
+An `is-unique` constraint MUST define only one `@key-field`. 
+
+If the evaluation of the Metapath `@target` of the `is-unique`, then the computed key does not occur more than once within the same capability, its value will result to true.
```

Comment:
I changed the statement to,
 `If the evaluation of the Metapath ``@target`` of the ``<key-field>`` results in a true result, then ``@target`` does not occur more than once.`
 
I believe it reads a lot better.

Reply:
The evaluation of the target must result in a node result with a value, not a boolean result. This is because the result is used as the value for the key segment represented by the `key-field`. This text needs to be reoriented with this understanding.

Reply:
This was resolved with the most recent edits.